server/pkg/analytics/analytics.go

```diff
@@ -18,13 +19,15 @@ const (
 	endDate      = "today"
 	metrics      = "ga:totalEvents"
 	dimensions   = "ga:eventLabel"
+	filters      = "ga:eventCategory!=key1"
```

Comment:
This change is good for now!! Need to check whether we can delete the particular key from Google Analytics